delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/routes/FilesRoutes.scala

```diff
@@ -290,6 +290,32 @@ final class FilesRoutes(
                     }
                   }
                 }
+              },
+              pathPrefix("register-update") {
```

Comment:
Can be fixed in a later PR but we have a problem, it clashes with this endpoint:
https://bluebrainnexus.io/docs/delta/api/files-api.html#create-using-put

Reply:
Not in the update operation but for the create one, we will need to provide a way to "register" a file where the user leaves Delta generating the id

Reply:
Ah yeah this was intentional - can we do without the version where we generate the Id? Fewer endpoints => smaller API surface => less maintenance (tests, docs, code, etc)